ansible/playbooks/init-nitro.yml

```diff
@@ -0,0 +1,39 @@
+- name: Setup Nitro
+  hosts: all
+  become: yes
+  tasks:
+    - name: Enable nitro tools though yum
+      command: amazon-linux-extras enable aws-nitro-enclaves-cli
+
+    - name: Install packages
+      yum:
+        name:
+          - aws-nitro-enclaves-cli
+          - aws-nitro-enclaves-cli-devel
+        state: latest
+
+    - name: Set the enclave memory allocation
```

Comment:
I tried to stay as true to our old setup as I could while adding a reproducible end state.  So, I stuck with the text replacement version that we were using previously.  I am curious though if we should either (a) specify the entire allocator file and then copy it instead of doing the inline replacement xor (b) not change the allocator and instead leave the decision up to a more specific use case.  Thoughts?

Reply:
I think this is fine

Reply:
The editing of the allocator file seems to be pretty standard practice for the time being, even though it is quite clunky. Examle here for nitro on eks
https://docs.aws.amazon.com/enclaves/latest/user/kubernetes.html#create-lt